lightly/api/api_workflow_selection.py

```diff
@@ -16,6 +17,9 @@
     TagData,
 )
 
+if TYPE_CHECKING:
+    from numpy.typing import NDArray
+
```

Comment:
`NDArray` is only available since numpy `1.20`. To remain backwards compatible we only import it while type-checking.